src/components/PhotoSwipe/InfoDialog/index.tsx

```diff
@@ -28,8 +42,30 @@ export function FileInfo({
     metadata,
     exif,
     scheduleUpdate,
-}) {
+}: Iprops) {
     const appContext = useContext(AppContext);
+    const [location, setLocation] = useState<Location>(null);
+
+    useEffect(() => {
+        if (!location && metadata) {
+            if (metadata.longitude || metadata.longitude === 0) {
```

Comment:
This check should be like this, right? Also, we will check for latitude too, right?
```
metadata.longitude && metadata.longitude !== 0
```

Reply:
so, `longitude = 0` is a valid value but `if(metadata.longitude) ` will return `false` for it, so added an `or` condition to allow that.

>Also, we will check for latitude too, right?

if either both of the coordinates will be present or neither will be present, so just checking for one will do that job